Ical.Net/DataTypes/Duration.cs

```diff
@@ -119,8 +119,8 @@ public static Duration FromSeconds(int seconds) =>
     /// Parses the specified value according to RFC 5545.
     /// </summary>
     /// <exception cref="System.FormatException">Thrown if the value is not a valid duration.</exception>
-    public static Duration? Parse(string value) =>
-        (Duration?) new DurationSerializer().Deserialize(new StringReader(value));
+    public static Duration Parse(string value) =>
+        (Duration) new DurationSerializer().Deserialize(new StringReader(value))!; // throws if null
```

Comment:
Actually it doesn't throw in all cases as of today (https://github.com/ical-org/ical.net/blob/3c4e95b876d6ea0ed108310df4f4dcc36ba30089/Ical.Net/Serialization/DataTypes/DurationSerializer.cs#L73) although it should.